lib/assessment.js

```diff
@@ -4,23 +4,33 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
-
+var min = function min(nums) {
+  return Math.min(nums)
 };
 
+
 // Question 2
 // Write a function that takes zero or more arguments
 // and returns the sum of that set
 // assume that all the argument are numbers
-var sum = function sum() {
-
+var sum = function sum(addends) {
+  var adding = 0;
+  for (i = 0; i < addends.length; i++) {
+    adding += addends[i];
+  }
+  return adding;
```

Comment:
Now **this** looks more like it. There's just one problem, and it's the same as what I mentioned above.

To access the arguments passed into a function, irrespective of how many there are, you need to use the `arguments` array-like object.

If you drop in `argument` instead of `addends` everywhere in the body, and take `addends` out of the list of parameters, the code should work.